internal/init_test.go

```diff
@@ -13,3 +22,77 @@ func TestAccBundleInitErrorOnUnknownFields(t *testing.T) {
 	_, _, err := RequireErrorRun(t, "bundle", "init", "./testdata/init/field-does-not-exist", "--output-dir", tmpDir)
 	assert.EqualError(t, err, "failed to compute file content for bar.tmpl. variable \"does_not_exist\" not defined")
 }
+
+// This test tests the MLOps Stacks DAB e2e and thus there's a couple of special
+// considerations to take note of:
+//
+//  1. Upstream changes to the MLOps Stacks DAB can cause this test to fail.
+//     In which case we should do one of:
+//     (a) Update this test to reflect the changes
+//     (b) Update the MLOps Stacks DAB to not break this test. Skip this test
+//     temporarily until the MLOps Stacks DAB is updated
+//
+//  2. While rare and to be avoided if possible, the CLI reserves the right to
+//     make changes that can break the MLOps Stacks DAB. In which case we should
+//     skip this test until the MLOps Stacks DAB is updated to work again.
+func TestAccBundleInitOnMlopsStacks(t *testing.T) {
+	env := GetEnvOrSkipTest(t, "CLOUD_ENV")
+	if env == "gcp" {
+		t.Skip("MLOps Stacks is not supported in GCP")
+	}
+	tmpDir1 := t.TempDir()
+	tmpDir2 := t.TempDir()
+
+	w, err := databricks.NewWorkspaceClient(&databricks.Config{})
+	require.NoError(t, err)
+
+	// Create a config file with the project name and root dir
+	initConfig := map[string]string{
+		"input_project_name":                    "project_name",
+		"input_root_dir":                        "repo_name",
+		"input_include_models_in_unity_catalog": "no",
```

Comment:
UC tests are possible, but the test will require and the equivalent assertions will need make finer grained assumptions. Needing to set up schema, setting more input parameters like the schema name and catalog name and specifying a target deployment explicitly are a couple I noticed.

Given this level of dependencies on the details, I'd prefer to have the UC test in the MLOps repo itself, so that the tests are managed alongside the template changes.

Reply:
Kk makes sense, can we keep a UC initialization test then? We don't have to deploy/run any of the workflows, but good to make sure we can at least initialize the template